workflows/ts2fs.py

```diff
@@ -17,7 +26,7 @@ def f(x):
 
         return ts.sample_count_stat(sample_sets, f, len(sample_sets),
                                     span_normalise=False, windows='sites',
-                                    polarised=True, mode='site', strict=False)
+                                    polarised=is_polarised, mode='site', strict=False)
```

Comment:
This does not produce an unpolarized frequency spectrum (perhaps surprisingly) - if you set `is_polarised=False` then what you'll get here for *every* site is the number of samples. (See [the docs](https://tskit.dev/tskit/docs/stable/stats.html#polarisation))

To wrap the frequency spectrum, you should just wrap it after the fact.

Reply:
I added a function `_fold_fs()` to generate folded FS.